lib/framework/action.dart

```diff
@@ -937,6 +939,10 @@ enum ActionType {
   callExternalMethod,
   invokeHaptic,
   callNativeMethod,
+  branchLinkInit,
+  branchLinkValidate,
+  branchLinkCreateDeeplink,
+  branchLinkCreateDeeplinkWithShareSheet,
```

Comment:
do we need `branchLinkCreateDeeplinkWithShareSheet`

Reply:
Removed it